users/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+
+from core.models import TimeStampedModel
+
+class User(TimeStampedModel):
+    name         = models.CharField(max_length=20)
+    nickname     = models.CharField(max_length=20)
+    email        = models.EmailField(max_length=100, unique=True)
+    kakao_id     = models.EmailField(max_length=100, null=True)
```

Comment:
`kakao_id`는 int 값으로 받게 됩니다.

```suggestion
    kakao_id = models.IntegerField(max_length=100, null=True)
```